steps/terraform.yaml

```
@@ -123,13 +126,27 @@ steps:
       workingDirectory: '$(System.DefaultWorkingDirectory)/$(buildRepoSuffix)/components/${{ parameters.component }}'
       environmentServiceName: ${{ parameters.serviceConnection }}
       runAzLogin: true
-      commandOptions: >
-        -out tfplan-$(tfPlanName)
-        -var env=${{ parameters.environment }}
-        -var builtFrom=$(Build.Repository.Name)
-        -var product=${{ parameters.product }} ${{ parameters.planCommandOptions }}
-        -var-file $(System.DefaultWorkingDirectory)/$(buildRepoSuffix)/environments/${{ parameters.environment }}/$(tfVarsName).tfvars
       publishPlanResults: "$(tfPlanName)"
+      ${{ if eq(parameters['terraformVarFile'], '') }}:
```

it's a vars file plural
```suggestion
      ${{ if eq(parameters['terraformVarsFile'], '') }}:
```

could also just be

```suggestion
      ${{ if eq(parameters['tfvarsFile'], '') }}:
```

---

Done